user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md

```diff
@@ -4,23 +4,25 @@ uid: DashboardStepper
 
 # Stepper
 
-Available from DataMiner 10.3.10/10.4.0 onwards.
+Available from DataMiner 10.3.10/10.4.0 onwards<!-- RN 37200 -->.
 
-This visualization takes a stateful object manager instance or object manager definition as data input and displays their respective states. This component simplifies complex workflows by breaking them down into distinct numbered or labeled steps.
+This component is used to guide the user through a workflow by splitting it up into different numbered or labeled steps. It indicates the progress through the workflow by showing the past steps, current step, and future steps. The component uses a stateful DOM instance or DOM definition (i.e. a DOM instance or DOM definition that contains states) as data input.
 
 To configure the component:
 
-1. Apply a data feed. See [Applying a data feed](xref:Configuring_dashboard_components#applying-a-data-feed).
+1. Apply a DOM definition or DOM instance data feed. See [Applying a data feed](xref:Configuring_dashboard_components#applying-a-data-feed).
 
-   - When applying a DataMiner object manager (DOM) definition, the component will display states along the typical path of that definition, i.e. the happy path. This path illustrates the states an instance would undergo following the standard workflow.
+   You can apply a DOM definition or DOM instance directly, or use e.g. a feed based on a table component, so that the stepper will display the information linked to the item selected in the table component.
+
+   - When a DOM definition is applied, this will be used to display states along the typical path of the definition, i.e. the "happy path". This path illustrates the states an instance would undergo following the standard workflow.
 
      > [!NOTE]
      > DOM definitions can only be applied to the stepper component within the Low-Code Apps module.
 
-   - When applying a DOM instance, the component will display the complete history of each state the instance has traversed until its current state. Based on the current state, it calculates and displays the remaining states along the happy path.
+   - When a DOM instance is applied, this will be used to display the complete history of each state the instance has traversed until its current state. In this case, the happy path is displayed based on the current state, which may not be the most happy path based on the DOM definition.
```

Comment:
> which may not be the most happy path based on the DOM definition.

I find this a little confusing, the happy path based on the current state of an instance will never be the same as the happy path you see for a definition, it will be a subset of that path where the start of the path isn't the first possible state but the current state the instance is in.

Reply:
The difference is that, since there isn't a current state, future states and past states in the case of a definition, it will always show the same happy path, i.e. the easiest path between the first possible state and the last possible state. Since an instance does have a current state and the happy path is the easiest path between the current state and the last possible state, this happy path will never be the same as the happy path shown for a definition. I think the second explanation is the best, but maybe you can say something like 'followed by the future states the instance will go through if it were to follow the normal flow from here'.

Reply:
How about this? 

```suggestion
- When a DOM instance is applied, this will be used to display the current state, preceded by the states the instance has traversed until its current state, and followed by the future states it will go through if it follows the normal flow.
```